OTPKitDemo/Features/OriginDestination/Components/OriginDestinationSheetView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  OriginDestinationSheetView.swift
+//  OTPKitDemo
+//
+//  Created by Hilmy Veradin on 25/06/24.
+//
+
+import MapKit
+import SwiftUI
+
+struct OriginDestinationSheetView: View {
```

Comment:
Can you add a short doc comment to the start of this view that explains what this does?